### PR TITLE
add mobile challenge

### PR DESCRIPTION
**Pull request by nicroth**

### CODE REVIEW

File: mobile.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Create an implementation of a **Document Uploader** app that meets the following requirement:
+
+* Allow a user to input/set their `Customer ID`
+	* No true login system is required, but bonus points if you want to make one.
+* Allow a user to take a photo of a physical document using the mobile device's camera
+	* Ensure the document's orientation/rotation is correct
+	* Allow the user to preview the photo before submitting
+* Upload the document to a web service, attaching the customer information
+* Give the user a confirmation that the document has been uploaded when the process is complete tell them that Avant will be reviewed by a verifications specialist within 24hrs.
```
Comment (fledman) [inline mobile.md]: I think you need to reword this line, maybe:

```
When the document has been uploaded, show the user a confirmation message specifying that Avant will review their submission within 24hrs.
```

Comment (nicroth) [inline mobile.md]: fixed in 65ddb33 - thanks!

File: mobile.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+* Ensure a user understands where they are in the process flow throughout
+* Ensure any network errors are handled properly and the user never gets "stuck" in the app
+* Bonus (not required)
+	* Allow the user to upload multiple documents
+	* Allow the user to view their uploaded documents
+	* Allow user to pick or tag additional data to documents such as:
+		* `Document Name` 
+		* `Document Type` (such as 1099, W2, Driver’s License, Voided Check)
+	* Design polish 
+	* Customer delighting features
```
Comment (ArtisanalPickle) [inline mobile.md]: could be a rabbit hole / timesink. I think people may feel prompted to spend too much time on here when we'd rather move them through the hiring process

File: mobile.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	* Create a simple web interface to view documents grouped by `Customer ID`
```
Comment (ArtisanalPickle) [inline mobile.md]: is this really relevant to the _mobile_ challenge? I think ultimately we will even want to build out our own Parse app or whatever so they don't have to worry about any of this stuff

File: mobile.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Mobile Challenge
+This challenge may be completed using any native mobile environment of your choice. Building the app for iOS (Objective-C/Swift) or Android (Java) is preferred, but please use whatever you are most comfortable with. For the web service, you may use any tool you are comfortable with. We suggest something simple, such as Parse. This challenge is designed to be open-ended.
+
+At Avant Credit, we provide unsecured personal loans, backed by proof that a customer has a steady income stream.  To verify that income stream or other vital customer information, our customers send us documents, proving that income (like W2 forms, 1099s, voided checks, utility bills, etc.).  To make the process of getting those documents to us faster and easier for our customers, we would like to build a mobile document uploader. The app will allow a customer to log in, take a picture of their documents using their phone, and then an Avant Credit verifications specialist will review the documents.
+
+Create an implementation of a **Document Uploader** app that meets the following requirement:
```
Comment (ArtisanalPickle) [inline mobile.md]: "requirements"

**Comment by ArtisanalPickle** [comment]
lgtm
